perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -59,6 +59,11 @@
                   'api_namespace: runs API based benchmarking for namespace '
                   'operations.')
 
+flags.DEFINE_enum('cli_size', 'normal',
+                  ['normal', 'large'],
+                  'size of the cli tests. Normal means half small and half \n'
+                  'large files. large means all large files each with 1GB.')
```

Comment:
"large files" has two fairly different meanings here, yes?

For 'normal', a "large file" is 32MB.  For 'large', a "large file" is 1GiB.